components/status_dropdown/status_dropdown.tsx

```diff
@@ -398,6 +400,9 @@ export default class StatusDropdown extends React.PureComponent<Props, State> {
 
         const customStatusComponent = this.renderCustomStatus(isStatusSet);
 
+        const menuAriaLabelValue = this.props.status;
+        const menuAriaLabeltext = intl.formatMessage({id: 'status_dropdown.menuAriaLabel.text', defaultMessage: 'user status is set to ' + menuAriaLabelValue + ' you can change status by opening'});
```

Comment:
The exact text still needs to be finalized, but to add a value like the status into a translated piece of text like this, you need to use the `values` option to do that so it'd look like
```suggestion
        const menuAriaLabeltext = intl.formatMessage({
            id: 'status_dropdown.menuAriaLabel.text',
            defaultMessage: 'user status is set to {status} you can change status by opening',
            values: {status: this.props.status},
        });
```